15/code-monk08/app.py

```diff
@@ -0,0 +1,59 @@
+#run export FLASK_ENV=development before running flask run
+import datetime, requests
+from flask import Flask, render_template, request, redirect
+from flask_sqlalchemy import SQLAlchemy
+
+key = open("apikey.txt","r").readline()
```

Comment:
use a with statement?

Reply:
yes, I use with statement these days.. PR is from April 12